.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+
+
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.7
+
+    steps:
+      - checkout
+      
+      - run:
+          name: install package and dependencies
+          command: |
+            python3 -m venv venv
```

Comment:
I think otherwise you have to sudo pip install? this was copied from an example on circleci's site. I'll check after it's in master - circleci doesn't seem to be picking it up from the branch.

Reply:
You could also switch to our new docker container registry and use a pre configured image.

Reply:
I'd like the requirements to be installed according to the files in the repo, so I don't think there's anything that we'd benefit from prebuilding here. we could potentially cache some eggs or something though, using the caching feature.